plaidml2/bridge/keras/backend_test.py

```diff
@@ -1071,49 +1090,88 @@ def testConv2dTranspose(self, b, x, k, os, st, pd, df, dr):
         ]
 
     @opTest([[m(1, 3, 3, 1), m(1, 3, 3, 1) - 2]], skip_tensorflow=True, skip_theano=True)
-    @unittest.skip('TODO: convert to EDSL')
+    #@unittest.skip('TODO: convert to EDSL')
     def testDefractLong(self, b, x, k):
-        f = ('function (I[N, L0, L1, CI], K[LK0, LK1, CO, CI]) -> (O) {\n' +
-             '  O[n, x0, x1, co: 1, 5, 5, 1] = +(I[n, (x0 + k0 - 1)/2, (x1 + k1 - 1)/2, ci]' +
-             ' * K[2 - k0, 2 - k1, co, ci]);\n}')
-        return [
-            tile.Operation(f, [('I', x), ('K', k)], [('O', tile.Shape(x.shape.dtype,
-                                                                      (1, 5, 5, 1)))],
-                           name='DefractTest').sole_output()
-        ]
+
+        I = x.tensor
+        K = k.tensor
+        N = edsl.TensorDim()
+        L0 = edsl.TensorDim()
+        L1 = edsl.TensorDim()
+        CI = edsl.TensorDim()
+        LK0 = edsl.TensorDim()
+        LK1 = edsl.TensorDim()
+        C0 = edsl.TensorDim()
+        C1 = edsl.TensorDim()
```

Comment:
Since the output is a fixed size and you're therefore not calling `bind_dims`, you also don't need to create all these `TensorDim`s.